src/SonarScanner.MSBuild/SonarScanner.MSBuild.csproj

```diff
@@ -6,7 +6,7 @@
     <RollForward>LatestMajor</RollForward>
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netcoreapp2.0|AnyCPU'">
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Nice catch! Indeed this would never happen.

Looking at it more closely I think it was not configured in a clear way:
```
The compiler command line accepts values greater than 4 to enable [warning wave warnings](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves). However, the .NET SDK sets the WarningLevel to match the AnalysisLevel in your project file.
```
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings#warninglevel

I imagine that the condition was there to work on the local machines. I'll try to update it to run for dotnet core in general.